src/scrapers/base-scraper-with-browser.js

```diff
@@ -94,7 +94,8 @@ class BaseScraperWithBrowser extends BaseScraper {
   async navigateTo(url, page) {
     const pageToUse = page || this.page;
     const response = await pageToUse.goto(url);
-    if (!response || response.status() !== OK_STATUS) {
+
+    if (response !== null && response.status() !== OK_STATUS) {
```

Comment:
this should work since in puppeteer documentation they mention that when changing hash a null result is returned. but if they will return `undefined` here for any reason we will have unhandled exception.

@eshaham please review previous commit in this pr and the comment I wrote and advice how to proceed.